### PR TITLE
various updates

### PR DESCRIPTION
**Pull request by bmfmancini**

### CODE REVIEW

File: Install-Under-CentOS_LAMP.md [hunk 1]
```diff
@@ -401,5 +408,12 @@ documentation on how to make your SELinux policy right.
 **Note:** If you installed Cacti out of `/var/www/html` make sure you fix up
 all SELinux context and permissions.
 
+### Use setup wizard script for an interactive installation
```
Comment (netniV) [inline Install-Under-CentOS_LAMP.md]: Whilst the script is useful for people to try out, I'm not sure it should be included in the official documentation.

File: Installing-Under-Ubuntu-Debian.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+```console
+apt-get update
+apt-get install -y apache2 rrdtool mariadb-server snmp snmpd php7.0 php-mysql php7.0-snmp php7.0-xml php7.0-mbstring php7.0-json php7.0-gd php7.0-gmp php7.0-zip php7.0-ldap php7.0-mc
+```
+
+### Downloading the Cacti software
+
+Once the OS packages are installed we need to download the Cacti files you can do this by using the git command
+&nbsp;
```
Comment (netniV) [inline Installing-Under-Ubuntu-Debian.md]: Inline HTML is bad.

File: Installing-Under-Ubuntu-Debian.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+$database_username = 'cactiuser';
+$database_password = 'cactiuser';
+$database_port     = '3306';
+$database_retries  = 5;
+$database_ssl      = false;
+$database_ssl_key  = '';
+```
+
+The system is now ready to go just surf to yourip/cacti to start the cacti initalization
+&nbsp;
```
Comment (netniV) [inline Installing-Under-Ubuntu-Debian.md]: More inline HTML

**Comment by bmfmancini** [comment]
changes commited

File: Install-Under-CentOS_LAMP.md (hunk 1 above)
Comment (netniV) [reply Install-Under-CentOS_LAMP.md]: This link has now been removed so can be considered resolved

**Comment by netniV** [comment]
I have now committed this but I will but updating a load of the files to be more in line with what we expect.

**Comment by netniV** [comment]
Thanks for your continued contribution!

**Comment by netniV** [comment]
I would recommend that you pull the latest changes in before attempting any more updates as there have been a few adjustments resulting in large changes to around 10 of the pages.

**Pull request merged**